.github/workflows/weekly.yml

```diff
@@ -1,9 +1,6 @@
 name: Lucky CLI Weekly CI
 
 on:
-  push:
-    branches: [main]
-  pull_request:
```

Comment:
This was causing this to run on every single commit. It should be a separate PR, but I needed it to quiet down with all the testing.